game/scripts/npc/abilities/earthshaker_aftershock.txt

```diff
@@ -17,7 +17,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityDuration"                                     "0.6 0.9 1.2 1.5 1.8 2.1"
+    "AbilityDuration"                                     "0.6 0.9 1.2 1.5 1.5 1.5" //OAA
```

Comment:
Not like I've been keeping track of OAA developments, but the `//OAA` comment shouldn't be necessary unless the vanilla values are modified right?

Reply:
Yeah, //OAA is not needed here.